src/parser/jobs/brd/modules/index.js

```diff
@@ -15,7 +14,6 @@ export default [
 	OGCDDowntime,
 	PitchPerfect,
 	SongUptime,
-	StraightShot,
```

Comment:
Is SS gonna be making a come-back soon? If not, can probs nuke the file for it too.

Reply:
It's not, I'll get rid of it.  Didn't want to remove it without talking about it first.